sigs/sig-marketing/social-media-guidelines.md

```diff
@@ -0,0 +1,50 @@
+**<span style="text-decoration:underline;">Assets</span>**
+
+[Argo Proj Social Content Calendar](https://docs.google.com/spreadsheets/d/1ysidts6KNAr2OKS1L4bgMToUFuAmO1xQ__tTQf4kPWk/edit?usp=sharing) (to prevent duplicate posting)
+
+**<span style="text-decoration:underline;">Guidelines*</span>**
+
+_Access_
+
+
+
+* Members of sig-marketing will have access to Argo Proj Social Accounts (LinkedIn, Twitter, and Youtube)
+* Those with access can post as they see fit, keeping the below guidelines in mind.
+* For posts that do not follow guidelines: 2 members from **_different_** organizations must agree that a post should be taken down and then it will be deleted.
+
+_Objectives_
+
+
+
+* Raising awareness of events and talks about/for the community 
+    * Includes: ArgoCon, KubeCon, GitOpsCon, Argo Contributor Summit, Workshops, Certifications, Community Events
+* Empowering the DevOps community to use and leverage Argo Projects
+* Conversion: Communicating with the existing Argo community on new features and updates
+
+_LinkedIn/Twitter Posts_
+
+
+
+* Maintainer updates
+    * Release Notes
+    * New Argoproj Blogs
+    * New Maintainers/Approvers
+* Events
+    * Event milestones
+    * Awareness around Argo Talks
+    * Visiting Argo Booth
+* Community Blogs
+    * Tips and Tricks, Best Practices Guide
+    * No vendor product blogs as it relates to Argo, but vendor blogs **_about_** Argo are OK
```

Comment:
We should encourage post from everyone, no matter if they work for a vendor or not, but blogs/tweets need to be neutral, e.g. a promoted blog or video can't have one section about Argo and another section about a product offering.